gradle.properties

```diff
@@ -17,12 +17,12 @@ junitVersion=4.12
 lombokVersion=1.18.4
 mockitoVersion=1.10.19
 scalaVersion=2.11.8
-shadowGradlePlugin=2.0.3
+shadowGradlePlugin=6.1.0
 slf4jApiVersion=1.7.25
 
 # Version and base tags can be overridden at build time.
 connectorVersion=0.10.0-SNAPSHOT
-pravegaVersion=0.10.0-2825.570df6c-SNAPSHOT
+pravegaVersion=0.10.1-2985.8ed3993-SNAPSHOT
```

Comment:
Any reason for choosing this commit point or is it the latest of `r10`?

Reply:
I had to change the artifact version because the previous one was from jCenter, and this does not exist anymore. The one I put is almost the last one for `r0.10`, but now I just updated it with the last one in `r0.10`.